core/src/components/radio/radio.ionic.scss

```diff
@@ -0,0 +1,182 @@
+@use "../../themes/ionic/ionic.globals.scss" as globals;
+@use "./radio.common";
+
+// Ionic Radio
+// --------------------------------------------------
+
+:host {
+  --color: #{globals.$ionic-color-neutral-500};
+  --color-checked: #{globals.ion-color(primary, base)};
+  --border-width: #{globals.$ionic-border-size-025};
+  --border-style: #{globals.$ionic-border-style-solid};
+  --border-radius: #{globals.$ionic-border-radius-full};
+  --focus-ring-color: #{globals.$ionic-state-focus-1};
+  --focus-ring-width: #{globals.$ionic-border-size-050};
+
+  @include globals.typography(globals.$ionic-body-md-regular);
+  min-height: globals.$ionic-scale-1200;
+
+  color: globals.$ionic-color-neutral-1200;
+}
+
+// Radio Color
+// --------------------------------------------------
+
+:host(.ion-color.radio-checked) .radio-icon {
+  border-color: globals.current-color(base);
+
+  background-color: globals.current-color(base);
+}
+
+// Radio Label
+// ----------------------------------------------------------------
+
+:host(.in-item) .label-text-wrapper {
+  @include globals.margin(globals.$ionic-space-250, null, globals.$ionic-space-250, null);
+}
+
+:host(.in-item.radio-label-placement-stacked) .label-text-wrapper {
+  @include globals.margin(globals.$ionic-space-250, null, globals.$ionic-space-400, null);
+}
+
+:host(.in-item.radio-label-placement-stacked) .native-wrapper {
+  @include globals.margin(null, null, globals.$ionic-space-250, null);
+}
+
+// Radio Native Wrapper
+// ----------------------------------------------------------------
+
+.native-wrapper .radio-icon {
+  width: globals.$ionic-scale-600;
+  height: globals.$ionic-scale-600;
+}
+
+// Ionic Radio Outer Circle: Unchecked
+// -----------------------------------------
+
+.radio-icon {
+  @include globals.margin(0);
+  @include globals.border-radius(var(--border-radius));
+
+  border-width: var(--border-width);
+  border-style: var(--border-style);
+  border-color: var(--color);
+
+  background-color: globals.$ionic-color-base-white;
+}
+
+// Ionic Radio Inner Circle: Unchecked
+// -----------------------------------------
+
+.radio-inner {
+  @include globals.border-radius(var(--inner-border-radius));
+
+  width: calc(32% + var(--border-width));
+  height: calc(32% + var(--border-width));
```

Comment:
Where does 32% come from?

Reply:
The designs show the inner circle to be 8px. It didn't feel correct to hard code it in and thought the best approach was to follow `md`. By using 32%, it calculates the size to be a very close number to 8px. It's like 8.0..px.